bukuserver/templates/bukuserver/home.html

```diff
@@ -4,6 +4,7 @@
 {% block head %}
   {{ super() }}
   {{ buku.close_if_popup() }}
+  {{ buku.brand_dbname() }}
```

Comment:
This also applies the same change to the Statistic page (since it extends `home.html`)